payment-manager/src/main/java/com/qpay/paymentmanager/service/impl/PaymentServiceImpl.java

```diff
@@ -33,6 +44,20 @@ public WalletEntity makePayment(final WalletPayment walletPayment) {
         final var updatedToWalletBalance = toWallet.getBalance().add(walletPayment.amount());
         updateBalance(updatedToWalletBalance, toWallet);
 
+        final var notificationClient = NotificationClient.builder()
+                .amount(walletPayment.amount())
+                .build();
+
+        final WebClient webClient = webClientConfig.getWebClient();
+        executorService.execute(() -> webClient
+                .post()
+                .uri(webClientConfig.getBaseUrl() + NOTIFICATION_PATH)
+                .bodyValue(notificationClient)
+                .retrieve()
+                .toBodilessEntity()
+                .block()
+        );
```

Comment:
51-59 lines have to be moved to NotificationClient file without executor service.

as a result you have to have only one line over here:
`executorService.submit/execute(() -> notificationClient.sendNotification(data));`